Assets/Scripts/Services/Interface/IDiscordService.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Assets.Scripts.Services.Interface
+{
+    public interface IDiscordService
+    {
+        void JoinViaDiscord(string roomID);
```

Comment:
JoinViaDiscord is only used within DiscordService. Could be made private if we don't intend to expose this method